File: api/src/main/java/com/ruby/rubia_server/core/enums/MessageStatus.java
package com.ruby.rubia_server.core.enums;

public enum MessageStatus {
    DRAFT("Rascunho"),
    SENDING("Enviando"),
    SENT("Enviado"),
    DELIVERED("Entregue"),
    READ("Lido"),
    FAILED("Falhou");
    
    private final String displayName;
    
    MessageStatus(String displayName) {
        this.displayName = displayName;
    }
    
    public String getDisplayName() {
        return displayName;
    }
}

File: api-dotnet/Rubia.Server/Enums/MessageStatus.cs
using System.ComponentModel;

namespace Rubia.Server.Enums;

public enum MessageStatus
{
    [Description("Rascunho")]
    Draft,
    
    [Description("Enviando")]
    Sending,
    
    [Description("Enviado")]
    Sent,
    
    [Description("Entregue")]
    Delivered,
    
    [Description("Lido")]
    Read,
    
    [Description("Falhou")]
    Failed
}